### DISCUSSION TITLE
Configure "Per block" CSS classes

### DISCUSSION THREAD

**Discussion by jmslbam** [ediamin/css-class-manager#2] [Ideas]
Hi! Thank you for this plugin, was looking for this for my clients.

Question: Is it possible to only have a specific set of CSS classes available per Block.

So that a CSS class that adjusts the `media-text block` doesn't show up at an Image block or so?

Kind regards!

**Comment by ediamin** [comment]
> Question: Is it possible to only have a specific set of CSS classes available per Block.

Hi @jmslbam I'm sorry, unfortunately it is not possible right now.

**Reply by jmslbam** [reply]
Thank you for the reply! No worries, maybe somewhere in the future :) will still use it!
